src/systems/userinput/userinput.js

```diff
@@ -157,21 +158,51 @@ function computeExecutionStrategy(sortedBindings, masks, activeSets) {
 
 AFRAME.registerSystem("userinput", {
   get(path) {
-    return this.frame && this.frame[path];
+    if (!this.frame) return;
```

Comment:
I changed this behavior because I did not want to return `false` when I meant `undefined`, but it probably doesn't matter because this only happens when `this.frame` doesn't exist, which only happens at all because we didn't choose the initialization order of components and systems